docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -423,17 +433,18 @@ ID             GRPC ENDPOINT        STATUS  LABELS
 ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
 pool-on-node-0  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-0  Online  10GiB     0 B        10GiB      0 B
 pool-on-node-2  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-2  Online  10GiB     0 B        10GiB      0 B
+
 kubectl mayastor get pools -n openebs --selector zone=eu-east-1
 ID             GRPC ENDPOINT        STATUS  LABELS
 ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
 pool-on-node-1  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-1  Online  10GiB     0 B        10GiB      0 B
 ```
 
-In this case, the replica of the volume will be placed on any of the two nodes i.e.
+In this case, the replica of the volume will be placed on any of the two pools i.e.
 - `pool-on-node-1` and `pool-on-node-2` or
 - `pool-on-node-1` and `pool-on-node-3` or
 - `pool-on-node-2` and `pool-on-node-3`
-as the storage class has `rack` as the value for `nodeHasTopoogyKey` that matches with the label key of the node.
+as the storage class has `rack` as the value for `nodeHasTopoogyKey` that matches with the label key of the pool.
```

Comment:
rack or zone?
typo in the 'nodeHasTopoogyKey', but should it be 'poolhasTopologyKey' instead?